pages/volunteer/tabs_template/developmentTab.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<div layout="row" flex class="oppia-foundation-white-background">
+  <md-content layout="column" flex="60" class="oppia-foundation-vertical-padding oppia-foundation-horizontal-padding oppia-foundation-white-background" layout-align-gt-md="center">
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla venenatis ante augue. Phasellus volutpat neque ac dui mattis vulputate.</p>
```

Comment:
?

Guessing you don't have text for this (and the other pages which use lorem ipsum). Can you send a request by email for that? We should be able to provide it.

Reply:
Don't `?` me, Sean :( I tried to come up with something but got writer's block!!

I have followed up the email thread with the request!

Reply:
Ah, sorry about that! Initially I thought it had been left in by mistake and that you'd forgotten to replace it. I only later realized that it was in the mock, but forgot to remove the first "paragraph" of the comment. Sorry!